ts/src/profiler.ts

```diff
@@ -267,6 +267,11 @@ export class Profiler extends common.ServiceObject {
       uri: '/profiles',
       body: reqBody,
       json: true,
+
+      // Default timeout for for request is 1 minute, but request to create
+      // profile is designed to hang until it is time to collect a profile
+      // (up to one hour).
+      timeout: 60 * 60 * 1000,
```

Comment:
Also: I thought we started to use parseDuration('1h')-like expressions instead of doing the direct math with millis?

Reply:
Yes -- Done.